build-system/pr-check.js

```diff
@@ -164,36 +196,76 @@ function determineBuildTargets(filePaths) {
 
 const command = {
   testBuildSystem: function() {
+    const startTime = startTimer('testBuildSystem');
     execOrDie('npm run ava');
+    stopTimer('testBuildSystem', startTime);
   },
   buildRuntime: function() {
+    let startTime = startTimer('buildRuntime: gulp clean');
     execOrDie(`${gulp} clean`);
+    stopTimer('buildRuntime: gulp clean', startTime);
+
+    startTime = startTimer('buildRuntime: gulp lint');
```

Comment:
Just noticed that execOrDie is a local function. Of course, this is a no-brainer.